ext/cuda/data_layouts_copyto.jl

```diff
@@ -16,18 +16,26 @@ function knl_copyto_linear!(dest, src, us)
     return nothing
 end
 
-if VERSION ≥ v"1.11.0-beta"
```

Comment:
We should keep this check for versions prior to 1.11.0-beta

Reply:
The main branch doesn't need this check, if you check out an older version of ClimaCore, it has this check, so removing it is still backwards compatible.

Reply:
(Same for the ones below)

Reply:
Just to confirm I understand, does ClimaCore main run on 1.10 and 1.11 independently of these checks? (So, removing this check is unrelated to upgrading to 1.11?)

Reply:
Yes

Reply:
Perfect, thank you!